client/src/Movies/Movie.js

```diff
@@ -11,7 +13,7 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const {id} = this.props.match.params
```

Comment:
Nice prop destructuring! (I can't remember if I helped so I genuinely mean this 😂 )
Another piece of code that would work is `const id = this.props.match.params.id` <-- Sets the variable `id` to the exact value.